lib/HttpClient.ts

```diff
@@ -28,22 +33,19 @@ type HttpRequestConfig = {
   headers?: Record<string, any>
 }
 
-export class HttpClient {
-  private static instance: Axios
+export class __HttpClient__ {
```

Comment:
A more functional way of doing this would be to just create functions for the different verbs that act on a certain type. Adding classes for these things kinda goes against the functional principles that the team is supposed to follow. We can have a look at alternative ways to achieve this at some point.

Reply:
Still think that this should be refactored to something other than class, but we can do that in the next iteration.